my-react-app/src/components/GetData/GetData.tsx

```diff
@@ -0,0 +1,97 @@
+//import { useState, useEffect } from "react";
+import DataItem from "./DataItem";
+import { useQuery, useQueryClient } from "@tanstack/react-query";
+import axios from "axios";
+
+type Data = {
+  id: number;
+  title: string;
+  year: string;
+  genre: string;
+  director: string;
+  actors: string;
+  language: string;
+  production: string;
+  website: string;
+};
+
+const GetData = () => {
+  //Using Axios & Tanstack query
+  const queryClient = useQueryClient();
+  const postQuery = useQuery({
+    queryKey: ["posts"],
+    queryFn: async () => {
+      const response = await axios.get("https://freetestapi.com/api/v1/movies");
+      queryClient.invalidateQueries(["posts"]);
```

Comment:
fix errors in IDE 

await queryClient.invalidateQueries({queryKey: ['posts']});